packages/react/src/VideoPlayer/VideoPlayer.stories.tsx

```diff
@@ -29,7 +41,7 @@ export default {
 export const Playground: StoryFn<typeof VideoPlayer> = args => (
   <VideoPlayer {...args}>
     <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+    <VideoPlayer.Track src="/example.vtt" default />
```

Comment:
@joshfarrant general comment about storybook.. the examples don't seem to work for me. 

https://github.com/user-attachments/assets/6b986c52-495e-48c9-9cc9-8aa05e2de9e2

Few other little things I noticed there:


- Timestamp is wrapping on both lines, probably due to broken video but we might want to double check it won't do this on longer videos (1hr+)?
  ![Screenshot 2024-07-10 at 10 36 17](https://github.com/user-attachments/assets/8f097172-0e30-4bd2-86b9-823674dffdd0)


- Could we add the poster to all of the examples please? Looks a little odd when not included.

  ![Screenshot 2024-07-10 at 10 36 08](https://github.com/user-attachments/assets/cc41c5f2-fc2f-40fb-a91c-1cce0484c7f1)

Reply:
Doh! Sorry, the timestamp wrapping is due to an issue in my previous commit. I reverted a previous commit from the command line but didn't test the change

I'll have a look at the examples as well, they work locally and (I think) they will work when deployed to the prod storybook, it looks like it's just these preview storybooks where they're not working. Leave that one with me.

Once I fix that the videos will look a bit less strange (you'll see the first frame instead of a white box) so we might not need posters there. 

If you pull the branch and run locally you'll be able to see that

Reply:
![image](https://github.com/user-attachments/assets/e6f67b2b-04e1-4305-a9da-3668bac08af7)

Timestamp when > 1 hour btw